src/app/views/experiment_page.js

```diff
@@ -0,0 +1,68 @@
+import View from 'ampersand-view';
+import dom from 'ampersand-dom';
+import mustache from 'mustache';
+import webChannel from '../lib/web_channel';
+
+export default View.extend({
+  template: mustache.render(`<section class="page" data-hook="experiment-page">
```

Comment:
I won't be impressed until we pre-process this with Jade before passing to ES6 strings, and Mustache.